src/main/java/graphql/servlet/internal/ApolloSubscriptionProtocolHandler.java

```diff
@@ -82,10 +87,12 @@ public void onMessage(HandshakeRequest request, Session session, WsSessionSubscr
                 break;
 
             case GQL_STOP:
+                keepAliveRunner.abort(session);
```

Comment:
Don't stop keep-alive here, as `GQL_STOP` is just to stop a single subscription stream